env/juwels/shell

```diff
@@ -4,12 +4,12 @@ module use /gpfs/software/juwels/otherstages
 module load Stages/2019a                
 module load StdEnv
 # For intel MPI
-module load Intel/2019.3.199-GCC-8.3.0  IntelMPI/2018.5.288 imkl/2019.3.199
-export FC=mpiifort CC=mpiicc CXX=mpiicpc
+#module load Intel/2019.3.199-GCC-8.3.0  IntelMPI/2018.5.288 imkl/2019.3.199
+#export FC=mpiifort CC=mpiicc CXX=mpiicpc
 
 # For ParaStation MPI 
-#module load  Intel/2019.3.199-GCC-8.3.0 ParaStationMPI/5.4 imkl/2019.5.281
-#export FC=mpifort CC=mpicc CXX=mpicxx
+module load  Intel/2019.3.199-GCC-8.3.0 ParaStationMPI/5.4 imkl/2019.5.281
+export FC=mpifort CC=mpicc CXX=mpicxx
```

Comment:
Are we more happy with this settings?

Reply:
Lately yes, for large mesh partitions (approximately >2000 CPUs) I have issues with IntelMPI when the communication structure is initialized by FESOM.

Reply:
Ok, let's keep your version then as a default.